src/Molecules/Filter/Filter.stories.jsx

```diff
@@ -28,8 +28,8 @@ export const filter = () => <Container>
   <Filter
     type="select"
     options={[
-      { name: '1', label: 'Choice 1' },
-      { name: '2', label: 'Choice 2' },
+      { value: '1', label: 'Choice 1' },
+      { value: '2', label: 'Choice 2' },
```

Comment:
What was the problem?
Can it happen on existing code using Filter?

Reply:
https://github.com/rgsystemes/lib-ui/blob/af2669ff9117434ce39343c7877004f3eb204b0c/src/Molecules/Filter/Filter.jsx#L75

It can happen if one is using the `Filter` component using `type="select"` with options not having `value` keys